### DISCUSSION TITLE
Where to find SCOS (Stream CoreOS) ISO for OKD 4.17.0?

### DISCUSSION THREAD

**Discussion by ron981** [okd-project/okd#2072] [Q&A]
Hi everyone,

I'm trying to set up an OKD cluster using version 4.17.0-okd-scos.0. According to the release information, I need Stream CoreOS version 417.9.202412022032-0.

Release details:
- OKD Version: 4.17.0-okd-scos.0
- Required SCOS Version: 417.9.202412022032-0
- Container Image: quay.io/okd/scos-release@sha256:a4c786872daa033c942a56ee437b52b1546abfe5f546754845b805a7cff6c0c4

I've already checked:
1. CentOS Mirrors
2. OKD GitHub releases
3. Fedora CoreOS builds
4. Various mirror sites

However, I couldn't find the bootable ISO file for this specific version. The container image is available on Quay.io, but I need the ISO for the initial cluster setup.

Could someone please point me to where I can download this ISO file? Or is there an alternative way to obtain it?

Thank you in advance for your help!

**Comment by JaimeMagiera** [comment]
There isn't an SCOS ISO. See here...

https://okd.io/docs/project/upgrade-notes/from-4-15/fcos-to-scos-migration#where-can-i-get-boot-artifacts-for-scos
